packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -578,10 +578,24 @@ const DatePickerInput = (props) => {
     }
     // decrement current valid date by 1 day, if not valid set date to yesterday instead
     if (event.key === '-' || event.key === '_') {
-      if (validDate) {
-        inputDate = DateUtil.decrementDateByDay(inputDate, DateUtil.ISO_EXTENDED_DATE_FORMAT);
-      } else {
-        inputDate = DateUtil.decrementDateByDay(DateUtil.getCurrentDate(), DateUtil.ISO_EXTENDED_DATE_FORMAT);
+      if (inputType === 0) {
```

Comment:
Instead of using 0, 1 and can you replace it similar it this line:
https://github.com/cerner/terra-framework/blob/91b1c58c141b3bcd6fae3875f3442807e2322a33/packages/terra-date-picker/src/DateInput.jsx#L273

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1999/commits/860c744021f979dc3fef08c89fc1551a75905869